src/frontend/js/components/CourseGlimpse/index.spec.tsx

```diff
@@ -92,4 +94,16 @@ describe('components/CourseGlimpse', () => {
     screen.getByText('Course 42');
     screen.getByText('Cover');
   });
+
+
+  it('shows does not include the Ref. label when the course code is not set', () => {
+    render(
+      <IntlProvider locale="en">
+        <CourseGlimpse context={contextProps} course={{ ...course, code: null }} />
```

Comment:
According to the `Course` type, `code` property cannot be null.

Reply:
Yep. The linter saw it too so I changed it.